system/lib/wasmfs/wasmfs.cpp

```diff
@@ -56,12 +56,17 @@ long __syscall_dup(long fd) {
 
   return fileTable.add(openFile.unlocked());
 }
-
-__wasi_errno_t __wasi_fd_write(__wasi_fd_t fd,
-                               const __wasi_ciovec_t* iovs,
-                               size_t iovs_len,
-                               __wasi_size_t* nwritten) {
-  if (iovs_len < 0) {
+enum Position { overwrite, preserve };
```

Comment:
```suggestion

enum class PositionHandling {
  overwrite,
  preserve
};

```
?

* Add "handling" to name, as this isn't a position itself but an operation on it.
* Enum class forces `A::B` usage (otherwise, `overwrite` and `preserve` are placed in the global namespace).
* Added spacing before and after, and internal indentation. But whatever clang-format wants here is good.

Reply:
Thanks, `clang-format` wants to compress it unfortunately :(.